SPIE2016-9913-16/9913-16.tex

```diff
@@ -0,0 +1,303 @@
+
+
+\section{Astropy}
+
+Python has become the de facto standard programming language for astronomy\cite{2000ASPC..216...59G,2006ASPC..351..343H,2006ASPC..351..497K,2011ASPC..442..425G,2012SPIE.8451E..02J} and is now the default choice for new projects and for teaching.
+The Astropy\cite{2013A&A...558A..33A} project grew out of a support mailing list in 2011 to become a very successful hub for the development of Python packages for astronomy.
+At the time of writing Astropy consists of approximately 105,000 lines of Python code.
+Astropy has two key approaches to drive adoption.
+The first is to place generic astronomy code into the \emph{core} package that is brought in via \texttt{import astropy} to provide a solid, well tested, base for astronomy software development; the second is to support and encourage the concept of \emph{affiliate} packages that are packages that use the \texttt{astropy} package and follow the same philosophy on distribution and openness.
+Together these aim to provide a \emph{one stop shop} for people trying to use Python in their astronomical research.
+
+\section{Synergies}
+
+The Astropy and LSST software projects are both attempting to write astronomical software in Python that will benefit the community.
+In LSST's case we are constrained by the requirements of MREFC funding to deliver a data management system on schedule and on budget and we can justify involvement with Astropy if it allows us to deliver the functionality without impacting schedule, or if opening up the LSST algorithms to the Astropy community encourages early adoption (through ease of installation) which can lead to more extensive testing.
+Additionally, the community will be encouraged to try out the LSST software sooner if it integrates well with Astropy and this will significantly enhance usage of the Level 3 processing system during LSST operations.
+In this section we look at the areas where collaboration between LSST and Astropy can enhance both projects.
+
+\subsection{Generalized Word Coordinate Systems}
+
+JWST and LSST have requirements for accurate world coordinate conversions that cannot be represented in the current FITS-WCS framework\cite{2013ASPC..475...49H,2002A&A...395.1077C,2015A&C....12..133T} and none of the current extensions handling focal plane distortions are suitably flexible\cite{2012SPIE.8451E..1MS,2012tpv}.
+This lack of flexibility in FITS-WCS has been a long standing issue and the Starlink AST library\cite{Berry201633} provides one solution to this problem that has been in use for nearly 20 years.
+For JWST a pure-Python equivalent, GWCS, is in development\cite{2016_P028_adassxxv} and is not yet part of Astropy but builds on the \texttt{astropy.modeling} framework.
+LSST expects to require world coordinate transformations to be performed in the C++ layer for performance reasons, and this may preclude direct adoption of GWCS by LSST, given the currently existing Python-based implementation.
```

Comment:
Given that GWCS has just been described as "in development", "existing Python-based implementation" comes off as a bit confusing.  Maybe "given the performance limitations of a pure-Python implementation", instead?